### PR TITLE
Added ATLAS S&C talk on ServiceX

### PR DESCRIPTION
**Pull request by mweinberg2718**

### CODE REVIEW

File: _data/people/mweinberg2718.yml [hunk 1]
```diff
@@ -6,3 +6,9 @@ institution: University of Chicago
 website:
 photo: /assets/images/team/Marc-Weinberg.jpg
 presentations:
+  - title: Delivery of columnar data to analysis systems
+    date: 2019-06-24
+    url: https://indico.cern.ch/event/765245/contributions/3470867/attachments/1869892/3076485/weinbergServiceX190626.pdf
+    meeting: ATLAS Software & Computing Week #62
```
Comment (henryiii) [inline _data/people/mweinberg2718.yml:12]:
```suggestion
    meeting: ATLAS Software & Computing Week \#62
```

I believe this is a comment in YAML.

Comment (pelmer) [reply _data/people/mweinberg2718.yml:12]: It came out literally. For special characters, I usually just put the whole text in quotes. I'll fit it.

Comment (pelmer) [reply _data/people/mweinberg2718.yml:12]: Actually, it came out literally for Brian Lin's version of this, which I put in quotes.

**Comment by bbockelm** [comment]
I'll fix it up directly - but note that this was missing the focus-area and the "project" tag.

Here's what I'll add:

```
--- a/_data/people/mweinberg2718.yml
+++ b/_data/people/mweinberg2718.yml
@@ -12,3 +12,5 @@ presentations:
     meeting: "ATLAS Software & Computing Week #62"
     meetingurl: https://indico.cern.ch/event/765245/
     location: New York City, New York
+    focus-area: doma, as
+    project: idds
```

I don't see any other missing tags, but hopefully this helps for the future!

**Pull request merged**